components/kyma-environment-broker/internal/storage/driver/postsql/events/events.go

```diff
@@ -0,0 +1,70 @@
+package events
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/dbmodel"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/postsql"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	Retention     time.Duration `envconfig:"default=72h"`
+	PollingPeriod time.Duration `envconfig:"default=1h"`
+}
+
+var ev *events
+
+type events struct {
+	postsql.Factory
+
+	log logrus.FieldLogger
+}
+
+func New(cfg Config, sess postsql.Factory, log logrus.FieldLogger) *events {
```

Comment:
@ralikio mentioned to add sync here

Reply:
done in https://github.com/kyma-project/control-plane/pull/2094/commits/0480a44e6747c8b9ea3867a6adc09efa175041ae